packages/runner-sdk/lib/action.ts

```diff
@@ -72,10 +72,7 @@ export abstract class NangoActionBase {
         this.environmentId = config.environmentId;
         this.providerConfigKey = config.providerConfigKey;
         this.runnerFlags = config.runnerFlags;
-
-        if (config.activityLogId) {
-            this.activityLogId = config.activityLogId;
-        }
+        this.activityLogId = config.activityLogId;
```

Comment:
config activityLogId of type `string` already. I think it is a leftover from a long time ago